src/Core/Usecase/Message/ReceiveMessage.php

```diff
@@ -163,6 +163,7 @@ private function createTargetedSurveyMessages($incoming_message)
 		$messageInSurveyState = $this->repo->get($surveyStateEntity->message_id);
 
 		// If we didn't find an outgoing message
+		// I'm not convinced this is right - if someone sends multiple incoming messages before the next outgoing one. But the survey isn't finished... shouldn't we add the messages to the original post?
```

Comment:
mmm. I think this might be weird if we add it to the contact's active post for the survey since it'd be linked to the survey?